build/templates/_converters.py.mako

```diff
@@ -10,8 +10,9 @@ import datetime
 def _convert_timedelta(value, library_type, scaling):
     scaled_value = value.total_seconds() * scaling
 
-    if not library_type == visatype.ViReal64 and not library_type == visatype.ViReal32:  # ctype integer types don't convert to int from float so we need to
-        scaled_value = int(scaled_value + 0.5)
+    # ctype integer types don't convert to int from float so we need to
```

Comment:
watch out, the rounding thing got reverted in the diff

Reply:
That was on purpose. There are some instances where we do not want to round.